devtools/databeta.sh

```diff
@@ -1,27 +1,51 @@
 #!/bin/sh
-# A script to compile a Dockerized data release:
-DATABETA=databeta-2021-03-30
+# A script to compile a Dockerized data release based on a user's local PUDL
+# data environment.
+
+# Name of the directory to create the data release archive in
+RELEASE_DIR=pudl-v0.4.0-2021-07-15
```

Comment:
does this have to be a hard coded date?

Reply:
No, I just haven't taken the time to (re-)learn how to add command line arguments to a bash script.

Reply:
ah okay! fair